### PR TITLE
[tests-only][full-ci] Add test for checking the response of ocm .well-known endpoint

### PR DESCRIPTION
**Pull request by nabim777**
## Description

In this PR there is an added test for checking the response of ocm .well-known endpoint
Scenario added:

```console
Scenario: check the well known ocm endpoint response
```

## Related Issue

- https://github.com/owncloud/ocis/issues/10726

## How Has This Been Tested?

- test environment:
- CI
- locally

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiOcm/ocmEndpoint.feature [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          "enabled": {
+            "const": true
+          },
+          "apiVersion": {
+            "const": "1.1.0"
+          },
+          "endPoint": {
+            "type": "string",
+            "pattern": "^%base_url%/ocm"
+          },
```
Comment (PrajwolAmatya) [inline tests/acceptance/features/apiOcm/ocmEndpoint.feature:34]:
```suggestion
          "endPoint": {
            "pattern": "^%base_url%/ocm"
          },
```
We can directly check the pattern.

**Comment by amrita-shrestha** [comment]
@ niraj what will happen if we hit this endpoint without ocm setup

File: tests/acceptance/features/apiOcm/ocmEndpoint.feature [hunk 2]
```diff
@@ -0,0 +1,84 @@
+@ocm
+Feature: OCM Well-Known Endpoint
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiOcm/ocmEndpoint.feature:2]:
```suggestion
Feature: ocm well-known URI
```

File: tests/acceptance/features/apiOcm/ocmEndpoint.feature [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  As a user
+  I want to verify the response of the .well-known/ocm endpoint
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiOcm/ocmEndpoint.feature:4]:
```suggestion
  I want to verify the response of well-known URI
```

**Review by amrita-shrestha — APPROVED** [review]
lgtm

**Pull request merged**